CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master (Unreleased)
 
+## 1.42.0 (2020-07-09)
+
+* Update RuboCop dependency to 0.87.0 because of changes to internal APIs. ([@bquorning][], [@Darhazer][])
```

Comment:
I added @Darhazer because half of my PR was him urging me make less of a mess :-)

Reply:
That I would need to add you as a co-author of half my contributions. Credits are not deserved here, you did a great job with the update!